### PR TITLE
Fixing small issues

### LINKED ISSUE #672: scoreboard_add should be really scoreboard_set

### LINKED ISSUE #687: [carpet suggestion] 2 very small and simple text changes
Just 2 aesthetic changes to feedback messages
1. Replacing "infinity mspt" with "∞ mspt"
2. Making counter names colored with their counter color,
so a red counter for example would have the word "red" typed with red colored letters

### LINKED ISSUE #690: Missing issues tag in fabric.mod.json.
The homepage tag redirects to issues instead. Both should be in use.

### PR DESCRIPTION
**Pull request by Ghoulboy78**
Fixes #690 
Fixes #687 
I know I was told one issue per pr, but these are really minor, and already had simple fixes which no one was bothered to implement.

### CODE REVIEW

File: src/main/java/carpet/script/api/Scoreboards.java [hunk 1]
```diff
@@ -115,6 +115,43 @@ public static void apply(Expression expression)
         // objective_add('counter')
 
         expression.addLazyFunction("scoreboard_add", -1, (c, t, lv)->
+        {
```
Comment (gnembon) [inline src/main/java/carpet/script/api/Scoreboards.java:118]: why you don't just use Expression.alias for that?

File: src/main/java/carpet/helpers/HopperCounter.java [hunk 2]
```diff
@@ -154,4 +157,44 @@ public long getTotalItems()
...
         return counter.values().stream().mapToLong(Long::longValue).sum();
     }
-}
+
+    private static String getPrettyColour(DyeColor colour){
+        return getPrettyColour(colour, "");
+    }
+
+    private static String getPrettyColour(DyeColor colour, String extra){
+        if (MaterialColor.WHITE.equals(colour.getMaterialColor())) {
```
Comment (gnembon) [inline src/main/java/carpet/helpers/HopperCounter.java:166]: This should be defined in a map, specifying appropriate formatting string given a colour.

Comment (gnembon) [reply src/main/java/carpet/helpers/HopperCounter.java:166]: And that mapping, as generic, could even be specified in WoolTool (I could then rename it to ColourTool)

File: src/main/java/carpet/helpers/HopperCounter.java [hunk 3]
```diff
@@ -43,6 +45,7 @@
     private HopperCounter(DyeColor color)
     {
         this.color = color;
+        this.prettyColour = getPrettyColour(color);
         // pubSubProvider = new PubSubInfoProvider<>(QuickCarpet.PUBSUB, "carpet.counter." + color.getName(), 0, this::getTotalItems);
```
Comment (gnembon) [inline src/main/java/carpet/helpers/HopperCounter.java:49]:
```suggestion
```

**Review by gnembon — CHANGES_REQUESTED** [review]

File: src/main/java/carpet/script/api/Scoreboards.java (hunk 1 above)
Comment (Ghoulboy78) [reply src/main/java/carpet/script/api/Scoreboards.java:118]: cos.. I didnt know it existed...

**Comment by Ghoulboy78** [comment]
wtf why does it think Im committing as Ghoulboy78-alt? thats so messed up...

**Comment by Ghoulboy78** [comment]
oh wait I inputted old email lol...

**Comment by replaceitem** [comment]
Depending on what gnembon thinks about my suggestion in #731, maybe the name `scoreboard_add` should be kept, in which case i will make a pr with `scoreboard_property()`

File: src/main/java/carpet/script/api/WorldAccess.java [hunk 4]
```diff
@@ -1027,6 +1026,8 @@ else if (item instanceof TridentItem || item instanceof SwordItem)
             return (_c, _t) -> Value.FALSE;
         });
 
+        expression.alias("place_item", "use_item");
```
Comment (gnembon) [inline src/main/java/carpet/script/api/WorldAccess.java:1029]: let me think about that, if that's really a more appropriate name.

File: src/main/java/carpet/mixins/ServerPlayNetworkHandler_scarpetEventsMixin.java [hunk 5]
```diff
@@ -240,8 +240,8 @@ private void onEntityInteract(PlayerInteractEntityC2SPacket playerInteractEntity
     ))
     private void onEntityAttack(PlayerInteractEntityC2SPacket playerInteractEntityC2SPacket_1, CallbackInfo ci)
     {
-        //todo add hit and hand in the future
-        PLAYER_ATTACKS_ENTITY.onEntityHandAction(player, playerInteractEntityC2SPacket_1.getEntity(player.getServerWorld()), null);
+        //todo add hit in the future
+        PLAYER_ATTACKS_ENTITY.onEntityHandAction(player, playerInteractEntityC2SPacket_1.getEntity(player.getServerWorld()), playerInteractEntityC2SPacket_1.getHand());
```
Comment (gnembon) [inline src/main/java/carpet/mixins/ServerPlayNetworkHandler_scarpetEventsMixin.java:244]: remove that - this all changed in 21w08a. No need to revise that now.

Again, too many small changes for one PR

File: src/main/java/carpet/script/api/WorldAccess.java (hunk 4 above)
Comment (gnembon) [reply src/main/java/carpet/script/api/WorldAccess.java:1029]: ok, its fine.

File: src/main/java/carpet/helpers/HopperCounter.java [hunk 6]
```diff
@@ -1,5 +1,6 @@
 package carpet.helpers;
 
+import carpet.utils.WoolTool;
```
Comment (gnembon) [inline src/main/java/carpet/helpers/HopperCounter.java:3]: This part is completely broken, throws NPE's and fails to display stuff. Did you run it?

File: src/main/java/carpet/script/api/Scoreboards.java [hunk 7]
```diff
@@ -114,11 +114,11 @@ public static void apply(Expression expression)
         // objective_add('lvl','level')
         // objective_add('counter')
 
-        expression.addLazyFunction("scoreboard_add", -1, (c, t, lv)->
+        expression.addLazyFunction("scoreboard_set", -1, (c, t, lv)->
```
Comment (gnembon) [inline src/main/java/carpet/script/api/Scoreboards.java:117]: yeah - lets wait for a PR to sort out scoreboard API, so don't change that

File: src/main/java/carpet/mixins/ServerPlayNetworkHandler_scarpetEventsMixin.java (hunk 5 above)
Comment (Ghoulboy78) [reply src/main/java/carpet/mixins/ServerPlayNetworkHandler_scarpetEventsMixin.java]: sorry.

File: src/main/java/carpet/helpers/HopperCounter.java (hunk 6 above)
Comment (Ghoulboy78) [reply src/main/java/carpet/helpers/HopperCounter.java:3]: k, fixed now

**Comment by DragonEggBedrockBreaking** [comment]
Some changes I suggest adding to this PR (in fabric.mod.json):

https://github.com/gnembon/fabric-carpet/blob/c5f666b95c0f4040883d40871eccfcacd2f7f3e6/src/main/resources/fabric.mod.json#L6
Change to `Fabric Carpet`, so that the list in your mods folder is the same as the list in MultiMC.

https://github.com/gnembon/fabric-carpet/blob/c5f666b95c0f4040883d40871eccfcacd2f7f3e6/src/main/resources/fabric.mod.json#L37
Remove this redundancy, as it is useless. It is not necessarily to remove it for a specific purpose, but more as a "why is it there, i don't need that" kind of thing.

**Comment by altrisi** [comment]
Hmmm, but this is Carpet Mod for Fabric, isn't it?

Fabric Carpet is more like the name of the repo IMO. You can also see in places such as the Readme that it is referenced as "Carpet Mod", not "Fabric Carpet".

**Comment by DragonEggBedrockBreaking** [comment]
But then it should be just `Carpet Mod`; `Carpet Mod in Fabric` is excessive, and especially because there is no (official) 1.14+ carpet mod for forge, or some other thing.

**Comment by Ghoulboy78** [comment]
Yes, but this helps distinguish it from like QuickCarpet. And before you say that QuickCarpet is also a fabric mod, it is also true that it is only ever branded as QuickCarpet, never as regular carpet. But idk about that too much.

**Comment by Ghoulboy78** [comment]
How bout we make a poll: 👍  for "Carpet mod for Fabric" , 👎 for "Fabric Carpet" and 💯 for thinking of something in between?

**Comment by DragonEggBedrockBreaking** [comment]
I cannot react with '100' so I did confused emoji instead.

**Comment by Ghoulboy78** [comment]
oh yes I'm stupid

File: src/main/java/carpet/script/api/WorldAccess.java [hunk 8]
```diff
@@ -1061,7 +1061,7 @@ else throw new InternalExpressionException("Attacking entity needs to be a livin
         expression.addLazyFunction("place_item", -1, (c, t, lv) ->
         {
             if (lv.size()<2)
-                throw new InternalExpressionException("'place_item' takes at least 2 parameters: item and block, or position, to place onto");
+                throw new InternalExpressionException("'use_item' takes at least 2 parameters: item and block, or position, to place onto");
```
Comment (gnembon) [inline src/main/java/carpet/script/api/WorldAccess.java:1064]:
```suggestion
                throw new InternalExpressionException("'place_item' takes at least 2 parameters: item and block, or position, to place onto");
```

**Pull request merged**